Makefile

```diff
@@ -1,55 +1,47 @@
-VIRTUALENV = virtualenv
 VENV := $(shell echo $${VIRTUAL_ENV-$$PWD/.venv})
-PYTHON = $(VENV)/bin/python
-DEV_STAMP = $(VENV)/.dev_env_installed.stamp
 INSTALL_STAMP = $(VENV)/.install.stamp
-TEMPDIR := $(shell mktemp -d)
 
 .IGNORE: clean
-.PHONY: all install virtualenv tests install-dev tests-once
+.PHONY: all install virtualenv tests tests-once
 
 OBJECTS = .venv .coverage
 
 all: install
-install: $(INSTALL_STAMP)
-$(INSTALL_STAMP): $(PYTHON) requirements.txt setup.py
+
+install: $(INSTALL_STAMP) pyproject.toml requirements.txt
+$(INSTALL_STAMP): pyproject.toml requirements.txt
+	python -m venv $(VENV)
```

Comment:
Will this recreate the virtual environment every time? I was thinking that the virtual environment step could be it's own command, then maybe `$(VENV)/bin/python` could be a dependency for `install`

Reply:
Apparently this command is idempotent, and it would have been reexecuted only when pyproject.toml requirements.txt would have changed, but you're right, it's cleaner to split